app/services/reporting/account_reuse_report.rb

```diff
@@ -173,6 +178,23 @@ def update_from_results(results:, total_registered:, total_proofed:)
             end
           end
         end
+
+        results.each_with_index do |details_section, section_index|
+          details_section.select { |details| details.num_entities >= 10 }.
+            reduce do |summary_row, captured_row|
+            # Delete any rows after the first captured_row (which becomes the summary_row)
+            details_section.delete(captured_row) if captured_row != summary_row
+            summary_row.update_details(
+              num_entities: "10-#{captured_row.num_entities}",
+              entity_type: summary_row.entity_type,
+              num_all_users: summary_row.num_all_users + captured_row.num_all_users,
+              all_percent: summary_row.all_percent + captured_row.all_percent,
+              num_idv_users: summary_row.num_idv_users + captured_row.num_idv_users,
+              idv_percent: summary_row.idv_percent + captured_row.idv_percent,
+            )
+          end
```

Comment:
I'd indent this more too
```suggestion
              # Delete any rows after the first captured_row (which becomes the summary_row)
              details_section.delete(captured_row) if captured_row != summary_row
              summary_row.update_details(
                num_entities: "10-#{captured_row.num_entities}",
                entity_type: summary_row.entity_type,
                num_all_users: summary_row.num_all_users + captured_row.num_all_users,
                all_percent: summary_row.all_percent + captured_row.all_percent,
                num_idv_users: summary_row.num_idv_users + captured_row.num_idv_users,
                idv_percent: summary_row.idv_percent + captured_row.idv_percent,
              )
            end
```

Reply:
Done!